ocamlbuild_plugin/ocamlbuild_cppo.ml

```diff
@@ -1,23 +1,22 @@
 
 open Ocamlbuild_plugin
 
+let insert_cppo_rules ext =
```

Comment:
Yeah, I think it's better. `cppo_rule` vs `cppo_rules` is differentiating enough IMO.